pkg/activator/handler/handler_test.go

```diff
@@ -476,26 +474,35 @@ func BenchmarkHandler(b *testing.B) {
 	configStore := setupConfigStore(&testing.T{}, logger)
 	ctx = configStore.ToContext(ctx)
 
-	for _, reps := range numReps {
-		respBody := strings.Repeat("Timmy Tucker was a friend of mine!", reps)
+	// bodyLength is in kilobytes.
+	for _, bodyLength := range [5]int{2, 16, 32, 64, 128} {
+		respBody := randomString(1024 * bodyLength)
 		fakeRT.RequestResponse.Body = respBody
-		for i := 2; i <= numReqs; i *= i {
-			b.Run(fmt.Sprintf("%06d-reqs-%07d-resp-len", i, len(respBody)), func(b *testing.B) {
-				for j := 0; j < i; j++ {
-					req := httptest.NewRequest(http.MethodGet, "http://example.com", nil)
-					req.Header.Set(activator.RevisionHeaderNamespace, testNamespace)
-					req.Header.Set(activator.RevisionHeaderName, testRevName)
-					req.Host = "test-host"
-					resp := httptest.NewRecorder()
-					handler.ServeHTTP(resp, req.WithContext(ctx))
-					if resp.Code != http.StatusOK {
-						b.Fatalf("RountripFailed responsecode = %d, want: StatusOK", resp.Code)
-					}
-					if got, want := resp.Body.Len(), len(respBody); got != want {
-						b.Fatalf("|body| = %d, want = %d", got, want)
-					}
+		b.Run(fmt.Sprintf("%03dk-resp-len", bodyLength), func(b *testing.B) {
+			req := httptest.NewRequest(http.MethodGet, "http://example.com", nil)
+			req.Header.Set(activator.RevisionHeaderNamespace, testNamespace)
+			req.Header.Set(activator.RevisionHeaderName, testRevName)
+			req.Host = "test-host"
```

Comment:
I actually think that creating and destroying body on each request is part of what we do... but if we want to capture _only_ proxying that's fine, I guess.